README.md

```diff
@@ -16,3 +16,39 @@ Check out our [getting started Guide](https://developers.livestorm.co/docs/getti
 ### You need a hand or want to contribute?
 
 We are here to help, feel free to drop an issue to our [Github repo](https://github.com/livestorm/livestorm-plugin), or to contact us directly at [plugins@livestorm.co](mailto:plugins@livestorm.co) 
+
+### How to run E2E tests
+
+You are contributing and you want to run the E2E tests before pushing your commit ?You have only have to make the followings steps:
+1. Set your Livestorm credentials in a cypress.env.json file
+
+    ```json
+    {
+
+        "TEAM_MEMBER_EMAIL": "email@livestorm.co", // The email of a team member of your organization
+
+        "TEAM_MEMBER_PASSWORD": "password" // The password of the latter
+    }
+    ```
+
+
+2. Set your plugin organization settings in a plugin/environments.json
```

Comment:
IMO this is not obvious where to create this file. It’s the one quoted here: https://github.com/livestorm/livestorm-plugin/pull/18/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR45